### PR TITLE
Addition to 275 release notes

### PR DESCRIPTION
**Pull request by agrawalreetika**
Test plan -
Documentation change

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by rohanpednekar** [comment]
Hi @highker, could you please approve and merge this PR? Thanks!

**Review by ajaygeorge — DISMISSED** [review]
LGTM

**Review by ericyuliu — APPROVED** [review]
LGTM.

File: presto-docs/src/main/sphinx/release/release-0.275.rst [hunk 1]
```diff
@@ -24,6 +24,10 @@ Spark Changes
 ______________
 * Add support for setting session properties for presto on spark.
 
+Hive Changes
```
Comment (ajaygeorge) [inline presto-docs/src/main/sphinx/release/release-0.275.rst:27]: can you move this above Hudi so that it is in order.

**Comment by arunthirupathi** [comment]
In the comment, can you please keep only no release notes are required.
Three backticks in the first line, NO RELEASE NOTE, three backticks in the next line.

` ` `
== NO RELEASE NOTE ==
` ` `

**Review by arunthirupathi — APPROVED** [review]

**Comment by arunthirupathi** [comment]
I have edited the PR description.

**Pull request merged**